Data/Sys/GameSettings/GALE01r2.ini

```diff
@@ -1136,17 +1097,33 @@ C2390CFC 00000021
 7C0802A6 00000000
 
 $Proper 16:9 Widescreen Support [Dan Salvato, mirrorbender]
```

Comment:
It's misleading to list these authors for the new version of the code. I'm not sure if you should list these authors in addition to you or just list you (how much of the old code is used in the current version of the code?), but it needs to be changed in some way so that we don't mislead people into thinking this is the code published by Dan Salvato and mirrorbender.

What's the difference between the old code and the new code, by the way?